CMakeLists.txt

```diff
@@ -75,6 +75,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CppUTestRootDirectory}/cmake/Module
 
 include("${CppUTestRootDirectory}/cmake/Modules/CppUTestConfigurationOptions.cmake")
 include(CTest)
+enable_testing()
```

Comment:
Why exactly in this line?
I mean it's between includes, is there any reason for this?

Reply:
Good catch! My intention was to keep it close to the related include, but yeah, this totally clutters the section.